main.js

```diff
@@ -205,57 +265,69 @@ define(function (require, exports, module) {
             $(window.document).off("mousedown", _documentClicked);
         }
     }
-    
+
     function _showSettings(e) {
         _hideSettings();
-        
+
         $settings = $(settingsHTML)
             .css({
                 right: 12,
                 top: $settingsToggle.position().top + $settingsToggle.outerHeight() + 12
             })
             .appendTo($panel);
-        
+        var selIndex = 0;
+        if (_prefs.get("useGFM")) selIndex = 1;
+        if (_prefs.get("useGFMCheckboxes")) selIndex = 2;
```

Comment:
These conditioned need to be grouped together. Otherwise, if the user edits the prefs file and sets `useGFM: false` and `useGFMCheckboxes:true`, the checked item will not match the behavior of the preview.

Something like:
```js
if (_prefs.get("useGFM")) {
    if (_prefs.get("useGFMCheckboxes")) {
        selIndex = 2;
    } else {
        selIndex = 1;
    }
}
```
(Also a minor stylistic nit - curly braces should be used around all expressions to match the style of the rest of the file)